x-pack/solutions/security/plugins/security_solution/public/onboarding/components/onboarding_body/cards/common/integrations/callouts/endpoint_callout.tsx

```diff
@@ -20,12 +20,12 @@ export const EndpointCallout = React.memo(() => {
   const { euiTheme } = useEuiTheme();
   const { docLinks } = useKibana().services;
   const {
-    telemetry: { trackLinkClick },
+    telemetry: { reportLinkClick },
```

Comment:
Why do we get the `reportLinkClick` from `useIntegrationContext` here, and from `useOnboardingContext` in the other callout (_agentless_available_callout.tsx_)? Which is the correct one to use?

Reply:
The two reportLinkClick from useIntegrationContext and useOnboardingContext are the same.
I planned to use reportLinkClick from useIntegrationContext in the integrations card, as it is used by cloud security, and wouldn't have access to OnboardingContext. For elsewhere in the onboarding cards, I'd use reportLinkClick from useOnboardingContext. 
With the logic above, yes this reportLinkClick should come from useIntegrationContext.
What do you think?